tests/tt_metal/tt_metal/unit_tests/compute/test_dropout_sfpu_compute.cpp

```diff
@@ -0,0 +1,258 @@
+            program,
+            "tt_metal/kernels/dataflow/reader_unary.cpp",
+            core,
+            DataMovementConfig{.processor = DataMovementProcessor::RISCV_1, .noc = NOC::RISCV_1_default});
+
+        KernelHandle unary_writer_kernel_id = CreateKernel(
+            program,
+            "tt_metal/kernels/dataflow/writer_unary.cpp",
+            core,
+            DataMovementConfig{.processor = DataMovementProcessor::RISCV_0, .noc = NOC::RISCV_0_default});
+
+        /*
+         * Set the parameters that the compute kernel will use.
+         */
+        vector<uint32_t> compute_kernel_args = {
+            num_tiles,
+            1,
+	    seed,
+	    int_probability,
+	    scale_factor
+        };
+
+        constexpr bool math_approx_mode = false;
+        const std::map<std::string, std::string> sfpu_defines = {
+            {"SFPU_OP_DROPOUT_INCLUDE", "1"},
+        };
+
+        KernelHandle eltwise_sfpu_kernel_id = CreateKernel(
+            program,
+            "/tests/tt_metal/tt_metal/test_kernels/compute/dropout_sfpu.cpp",
+            core,
+            ComputeConfig{
+                .math_approx_mode = math_approx_mode,
+                .compile_args = compute_kernel_args,
+                .defines = sfpu_defines,
+            }
+        );
+
+        /*
+         * Create source data and write to DRAM.
+         */
+        std::vector<uint32_t> src0_vec = create_constant_vector_of_bfloat16(
+            dram_buffer_size, const_bias);
+
+        tt_metal::detail::WriteToBuffer(src0_dram_buffer, src0_vec);
+
+        /*
+         * Configure program and runtime kernel arguments, then execute.
+         */
+        SetRuntimeArgs(
+            program,
+            unary_reader_kernel_id,
+            core,
+            {
+                src0_dram_buffer->address(),
+                static_cast<uint32_t>(src0_dram_buffer->noc_coordinates().x),
+                static_cast<uint32_t>(src0_dram_buffer->noc_coordinates().y),
+                num_tiles,
+            }
+        );
+
+        SetRuntimeArgs(
+            program,
+            unary_writer_kernel_id,
+            core,
+            {
+                dst_dram_buffer->address(),
+                static_cast<uint32_t>(dst_dram_buffer->noc_coordinates().x),
+                static_cast<uint32_t>(dst_dram_buffer->noc_coordinates().y),
+                num_tiles
+            }
+        );
+
+        tt_metal::detail::LaunchProgram(device, program);
+
+        /*
+         * Read the result and compare to a golden result. Record pass/fail
+         * and teardown.
+         */
+        std::vector<uint32_t> result_vec;
+        tt_metal::detail::ReadFromBuffer(dst_dram_buffer, result_vec);
+
+        auto transform_identity = [](const bfloat16 &a) {
+            return a;
+        };
+
+	// Unpack source and result vectors.
+        std::vector<bfloat16> src0_vec_bfloat16 = unpack_uint32_vec_into_bfloat16_vec(src0_vec, transform_identity);
```

Comment:
Minor: Fix indent

Reply:
Is there any script or command to format the files ? Not sure how much to indent .....

Reply:
We normally use clang-format to format files, but haven't enabled a global check/enforcement yet. This comment was specifically just because the line `// Unpack source and result vectors.` was misaligned with the surrounding lines and should have an additional indent to line it up.